Assets/Project/Scripts/MenuSelectScene/Settings/SettingsManager.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Project.Scripts.Utils.Definitions;
+using Project.Scripts.Utils.PlayerPrefsUtils;
+using UnityEngine;
+
+namespace Project.Scripts.MenuSelectScene.Settings
+{
+    public class SettingsManager : MonoBehaviour
```

Comment:
```suggestion
    public class SettingsManager
```
`MonoBehaviour` いらないかな

Reply:
コピペがバレる